waypoint.hcl

```diff
@@ -341,6 +346,11 @@ app "ssr-web" {
       when    = "after"
       command = ["node", "./scripts/purge-target-groups.mjs", "ssr-web"]
     }
+
+    hook {
+      when    = "after"
```

Comment:
did you want this to be "before"?

Reply:
I tried using a `before` hook in the `release` block, but waypoint complains when the block does not contain the `use` stanza: https://github.com/cardstack/cardstack/runs/6509127447?check_suite_focus=true

I tried `use = ""` as well, but waypoint then ignores any hooks that we defined in the `release` block: https://github.com/cardstack/cardstack/runs/6521301141?check_suite_focus=true

Using the `after` hook in the `deploy` block does allow the hooks to work properly: https://github.com/cardstack/cardstack/runs/6511150292?check_suite_focus=true